src/webserver.js

```diff
@@ -103,9 +103,9 @@ function initializeNodeBB(callback) {
 		},
 		function (next) {
 			async.series([
-				async.apply(meta.sounds.init),
-				async.apply(languages.init),
-				async.apply(meta.blacklist.load)
+				meta.sounds.addUploads,
+				languages.init,
+				meta.blacklist.load,
```

Comment:
Yes but I believe that's only because it was causing failed builds and nobody has had enough time to fix it yet.